code/datums/supply_packs/food.dm

```diff
@@ -205,7 +355,32 @@
 	containertype = /obj/structure/closet/crate/freezer
 	containername = "\improper Special ingredients crate"
 	group = "Food"
-	
+
+
+/datum/supply_packs/condiment
```

Comment:
you have not made a new path for this supply pack

Reply:
fixed